best-practices/README.md

```diff
@@ -232,8 +232,14 @@ CSS
   project-specific browser support that is needed.
 * Prefer `overflow: auto` to `overflow: scroll`, because `scroll` will always
   display scrollbars outside of macOS, even when content fits in the container.
+* [Create breakpoints](breakpoints) when the content "breaks," and is awkward or 
+  difficult to read,
+  * Avoid creating breakpoints that target specific devices
+  * Prefer `em` units instead of `px` for breakpoint values
+  * Start with a mobile breakpoint and work outwards using `min-width`/`min-height`
```

Comment:
Perhaps instead of specifying `mobile` here, we should remain agnostic and just say "smallest viewport" size?

Reply:
Love it. I'll update that now.